cppcache/integration/test/FunctionExecutionTest.cpp

```diff
@@ -26,10 +27,12 @@
 
 #include "framework/Cluster.h"
 #include "framework/Gfsh.h"
+#include "framework/config.h"
```

Comment:
Maybe don't access the config.h macros directly. They configure the framework. Add a class to the framework that provides and interface into specific framework values.